src/content/docs/en/guides/authentication.mdx

```diff
@@ -188,6 +188,124 @@ const session = await getSession(Astro.request);
 - [`auth-astro` on GitHub](https://github.com/nowaythatworked/auth-astro?tab=readme-ov-file#auth-astro)
 - [Auth.js documentation](https://authjs.dev/)
 
+## Better Auth
+
+Better Auth is a framework-agnostic authentication (and authorization) framework for TypeScript. It provides a comprehensive set of features out of the box and includes a plugin ecosystem that simplifies adding advanced functionalities.
+
+It supports Astro out of the box, and you can use it to add authentication to your astro project.
+
+### Installation
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install better-auth
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add better-auth
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add better-auth
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+For detailed setup instructions, check out the [Installtion Guide](https://www.better-auth.com/docs/installation).
+
+### Configuration
+
+Once you have installed and configured Better Auth in your project, you'll need to mount Better Auth handler in your Astro project.
+
+```ts title="page/api/auth/[...all].ts"
+import { auth } from "~/auth"; // import your Better Auth instance
```

Comment:
```suggestion
import { auth } from "../......"; // import your Better Auth instance
```
Can we make this a relative path instead of an import alias? copying this code won't work if people don't have that import configured.

This might mean, for the purposes of this example, we need a step where this is actually shown, creating a specific file to do so.